### PR TITLE
Draft: Testing the use of playwright instead of Cypress

### PR DESCRIPTION
**Pull request by AtharvChandratre**
@Betree this PR contains the code for enabling playwright in the project, and the scripts we used for testing the differences in run time for Cypress and Playwright.

### CODE REVIEW

File: .github/workflows/playwright.yml [hunk 1]
```diff
@@ -0,0 +1,27 @@
+name: Playwright Tests
+on:
+  push:
+    branches: [ main, master ]
+  pull_request:
+    branches: [ main, master ]
```
Comment (Betree) [inline .github/workflows/playwright.yml:6]:
```suggestion
    branches: [ main ]
  pull_request:
    branches: [ main ]
```

File: components/admin-panel/AdminPanelSection.js [hunk 2]
```diff
@@ -36,7 +36,6 @@ const ADMIN_PANEL_SECTIONS = {
   [HOST_DASHBOARD_SECTIONS.PENDING_CONTRIBUTIONS]: PendingContributions,
   [HOST_DASHBOARD_SECTIONS.EXPENSES]: HostDashboardExpenses,
   [HOST_DASHBOARD_SECTIONS.HOST_APPLICATIONS]: HostApplications,
-  [HOST_DASHBOARD_SECTIONS.PENDING_APPLICATIONS]: HostApplications,
```
Comment (Betree) [inline components/admin-panel/AdminPanelSection.js:39]: Looks unrelated

**Review by Betree — COMMENTED** [review]
There are many unrelated changes in this PR that make it difficult to review. Maybe it needs a rebase?

**Comment by AtharvChandratre** [comment]
@Betree  - https://github.com/opencollective/opencollective-frontend/pull/9448
